mtp_noms_ops/apps/settings/forms.py

```diff
@@ -264,3 +264,38 @@ def get_confirmation_query_string(self):
         query_dict = self.request.GET.copy()
         query_dict['prisons'] = prisons
         return urlencode(query_dict, doseq=True)
+
+
+class JobInformationForm(ApiForm):
+    job_titles = [
+        ('Evidence collator', 'Evidence collator'),
```

Comment:
the values should be wrapped as lazy translations. even though we might never turn on Welsh, it'll be so much more arduous to put them in later than at the time a feature is worked on

Reply:
also, the values have spaces, but in your template you have things like `{{ field.html_name }}-{{ value }}` which would be invalid HTML ids

Reply:
Good spot - hadn't really thought about the knock on effects of user friendly form values.